minor amendments to boyd_box

88hoddle opened the pull request:
Hi Steve, i found a slight error in my blockage code for boyd box. I should be ok now. There should be 2 files, boyd_box_operator.py and test_boyd_box_operator.py

regards

stoiver commented:
Hi Petar,


It looks like you deleted some files and then added them back into the repository. Unfortunately this will mess up the logging of changes to those files overtime.


I should be able to clean this up without too much problem. I'll give it a go on Monday.


Cheers

Steve


==============================
Stephen Roberts
Undergraduate Convenor
Mathematical Sciences Institute
Room 2005 P.A.P. Moran Building #26B
The Australian National University
Canberra, ACT 2601 AUSTRALIA
Ph: +61 2 61254445
CRICOS: 00120C

________________________________
From: Petar Milevski <notifications@github.com>
Sent: Saturday, 21 January 2017 10:08:56 AM
To: GeoscienceAustralia/anuga_core
Cc: Subscribed
Subject: [GeoscienceAustralia/anuga_core] minor amendments to boyd_box (#118)


Hi Steve, i found a slight error in my blockage code for boyd box. I should be ok now. There should be 2 files, boyd_box_operator.py and test_boyd_box_operator.py

regards

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/GeoscienceAustralia/anuga_core/pull/118

Commit Summary

  *   Hi Steve, i have added blockage to weir orifice trapezoid operator.
  *   Add files via upload
  *   Add files via upload
  *   Add files via upload
  *   Delete boyd_box_operator.py
  *   Delete boyd_pipe_operator.py
  *   Add files via upload
  *   Add files via upload
  *   Delete test_boyd_box_operator.py
  *   Add files via upload

File Changes

  *   M anuga/parallel/parallel_boyd_pipe_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-0> (70)
  *   M anuga/parallel/parallel_operator_factory.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-1> (9)
  *   M anuga/parallel/parallel_weir_orifice_trapezoid_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-2> (9)
  *   M anuga/structures/boyd_box_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-3> (66)
  *   M anuga/structures/boyd_pipe_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-4> (20)
  *   M anuga/structures/tests/test_boyd_box_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-5> (23)
  *   M anuga/structures/weir_orifice_trapezoid_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-6> (299)
  *   A boyd_pipe_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-7> (378)
  *   A parallel_boyd_pipe_operator.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-8> (341)
  *   A parallel_operator_factory.py<https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-9> (971)

Patch Links:

  *   https://github.com/GeoscienceAustralia/anuga_core/pull/118.patch
  *   https://github.com/GeoscienceAustralia/anuga_core/pull/118.diff

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/GeoscienceAustralia/anuga_core/pull/118>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFTlJHaZ19qmUuIYm_4FQi4bZ9BG0cnwks5rUT6IgaJpZM4Lp2Kh>.

88hoddle commented:
Hi Steve.

Yes I'm still trying to get my head around git.

I tried to push some changes via terminal and it would not do it. so then I
added the files via git and was in the wrong directory.

So I deleted them and then added then back into the correct directory.

Regards

On 22 Jan 2017 21:43, "Stephen Roberts" <notifications@github.com> wrote:

> Hi Petar,
>
>
> It looks like you deleted some files and then added them back into the
> repository. Unfortunately this will mess up the logging of changes to those
> files overtime.
>
>
> I should be able to clean this up without too much problem. I'll give it a
> go on Monday.
>
>
> Cheers
>
> Steve
>
>
> ==============================
> Stephen Roberts
> Undergraduate Convenor
> Mathematical Sciences Institute
> Room 2005 P.A.P. Moran Building #26B
> The Australian National University
> Canberra, ACT 2601 AUSTRALIA
> Ph: +61 2 61254445 <+61%202%206125%204445>
> CRICOS: 00120C
>
> ________________________________
> From: Petar Milevski <notifications@github.com>
> Sent: Saturday, 21 January 2017 10:08:56 AM
> To: GeoscienceAustralia/anuga_core
> Cc: Subscribed
> Subject: [GeoscienceAustralia/anuga_core] minor amendments to boyd_box
> (#118)
>
>
> Hi Steve, i found a slight error in my blockage code for boyd box. I
> should be ok now. There should be 2 files, boyd_box_operator.py and
> test_boyd_box_operator.py
>
> regards
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/GeoscienceAustralia/anuga_core/pull/118
>
> Commit Summary
>
> * Hi Steve, i have added blockage to weir orifice trapezoid operator.
> * Add files via upload
> * Add files via upload
> * Add files via upload
> * Delete boyd_box_operator.py
> * Delete boyd_pipe_operator.py
> * Add files via upload
> * Add files via upload
> * Delete test_boyd_box_operator.py
> * Add files via upload
>
> File Changes
>
> * M anuga/parallel/parallel_boyd_pipe_operator.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-0> (70)
> * M anuga/parallel/parallel_operator_factory.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-1> (9)
> * M anuga/parallel/parallel_weir_orifice_trapezoid_operator.py<
> https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-2>
> (9)
> * M anuga/structures/boyd_box_operator.py<https://github.
> com/GeoscienceAustralia/anuga_core/pull/118/files#diff-3> (66)
> * M anuga/structures/boyd_pipe_operator.py<https://github.
> com/GeoscienceAustralia/anuga_core/pull/118/files#diff-4> (20)
> * M anuga/structures/tests/test_boyd_box_operator.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-5> (23)
> * M anuga/structures/weir_orifice_trapezoid_operator.py<https://
> github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-6> (299)
> * A boyd_pipe_operator.py<https://github.com/GeoscienceAustralia/anuga_
> core/pull/118/files#diff-7> (378)
> * A parallel_boyd_pipe_operator.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-8> (341)
> * A parallel_operator_factory.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-9> (971)
>
> Patch Links:
>
> * https://github.com/GeoscienceAustralia/anuga_core/pull/118.patch
> * https://github.com/GeoscienceAustralia/anuga_core/pull/118.diff
>
> -
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118>, or mute the thread<
> https://github.com/notifications/unsubscribe-auth/AFTlJHaZ19qmUuIYm_
> 4FQi4bZ9BG0cnwks5rUT6IgaJpZM4Lp2Kh>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GeoscienceAustralia/anuga_core/pull/118#issuecomment-274323182>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AScWtguZzvLSwwEdlZlrFcXTbYRUv2zIks5rUzLtgaJpZM4Lp2Kh>
> .
>

88hoddle commented:
Thanks Steve

On 22 Jan 2017 21:43, "Stephen Roberts" <notifications@github.com> wrote:

> Hi Petar,
>
>
> It looks like you deleted some files and then added them back into the
> repository. Unfortunately this will mess up the logging of changes to those
> files overtime.
>
>
> I should be able to clean this up without too much problem. I'll give it a
> go on Monday.
>
>
> Cheers
>
> Steve
>
>
> ==============================
> Stephen Roberts
> Undergraduate Convenor
> Mathematical Sciences Institute
> Room 2005 P.A.P. Moran Building #26B
> The Australian National University
> Canberra, ACT 2601 AUSTRALIA
> Ph: +61 2 61254445 <+61%202%206125%204445>
> CRICOS: 00120C
>
> ________________________________
> From: Petar Milevski <notifications@github.com>
> Sent: Saturday, 21 January 2017 10:08:56 AM
> To: GeoscienceAustralia/anuga_core
> Cc: Subscribed
> Subject: [GeoscienceAustralia/anuga_core] minor amendments to boyd_box
> (#118)
>
>
> Hi Steve, i found a slight error in my blockage code for boyd box. I
> should be ok now. There should be 2 files, boyd_box_operator.py and
> test_boyd_box_operator.py
>
> regards
>
> ________________________________
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/GeoscienceAustralia/anuga_core/pull/118
>
> Commit Summary
>
> * Hi Steve, i have added blockage to weir orifice trapezoid operator.
> * Add files via upload
> * Add files via upload
> * Add files via upload
> * Delete boyd_box_operator.py
> * Delete boyd_pipe_operator.py
> * Add files via upload
> * Add files via upload
> * Delete test_boyd_box_operator.py
> * Add files via upload
>
> File Changes
>
> * M anuga/parallel/parallel_boyd_pipe_operator.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-0> (70)
> * M anuga/parallel/parallel_operator_factory.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-1> (9)
> * M anuga/parallel/parallel_weir_orifice_trapezoid_operator.py<
> https://github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-2>
> (9)
> * M anuga/structures/boyd_box_operator.py<https://github.
> com/GeoscienceAustralia/anuga_core/pull/118/files#diff-3> (66)
> * M anuga/structures/boyd_pipe_operator.py<https://github.
> com/GeoscienceAustralia/anuga_core/pull/118/files#diff-4> (20)
> * M anuga/structures/tests/test_boyd_box_operator.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-5> (23)
> * M anuga/structures/weir_orifice_trapezoid_operator.py<https://
> github.com/GeoscienceAustralia/anuga_core/pull/118/files#diff-6> (299)
> * A boyd_pipe_operator.py<https://github.com/GeoscienceAustralia/anuga_
> core/pull/118/files#diff-7> (378)
> * A parallel_boyd_pipe_operator.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-8> (341)
> * A parallel_operator_factory.py<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118/files#diff-9> (971)
>
> Patch Links:
>
> * https://github.com/GeoscienceAustralia/anuga_core/pull/118.patch
> * https://github.com/GeoscienceAustralia/anuga_core/pull/118.diff
>
> -
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub<https://github.com/
> GeoscienceAustralia/anuga_core/pull/118>, or mute the thread<
> https://github.com/notifications/unsubscribe-auth/AFTlJHaZ19qmUuIYm_
> 4FQi4bZ9BG0cnwks5rUT6IgaJpZM4Lp2Kh>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GeoscienceAustralia/anuga_core/pull/118#issuecomment-274323182>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AScWtguZzvLSwwEdlZlrFcXTbYRUv2zIks5rUzLtgaJpZM4Lp2Kh>
> .
>
